app/views/sessions/new.html.erb

```diff
@@ -1,12 +1,16 @@
-<h1>Login</h1>
-<%= form_tag sessions_path do %>
-  <div class="field">
-    <%= label_tag :email %>
-    <%= text_field_tag :email %>
-  </div>
-  <div class="field">
-    <%= label_tag :password %>
-    <%= password_field_tag :password %>
-  </div>
+<div id="login">
+  <div class="contiainer">
```

Comment:
fix typo, also `container` needs inside a `row`, and a `row` can have many `col-size-*`, then you insert the content. That way it will stay responsive across sizes, and you can always create another `container` inside the view, it will just add a bit more padding